lib/src/main/java/org/team100/lib/motion/components/OutboardLinearPositionServo.java

```diff
@@ -0,0 +1,87 @@
+package org.team100.lib.motion.components;
+
+import java.util.OptionalDouble;
+
+import org.team100.lib.controller.State100;
+import org.team100.lib.motion.LinearMechanism;
+import org.team100.lib.profile.Profile100;
+import org.team100.lib.telemetry.SupplierLogger;
+import org.team100.lib.telemetry.Telemetry.Level;
+
+/**
+ * Position control using duty cycle feature of linear mechanism
+ */
+public class OutboardLinearPositionServo implements LinearPositionServo {
```

Comment:
the word "outboard" in our code means "using the feedback controller in the motor controller hardware" which is not what you're doing here.

Reply:
Fixed